addons/mail/static/src/models/thread/thread.js

```diff
@@ -578,26 +613,25 @@ function factory(dependencies) {
         /**
          * Mark the specified conversation as read/seen.
          *
-         * @param {integer} messageId the message to be considered as last seen
+         * @param {integer} lastMessageId the message to be considered as last seen.
          */
-        async markAsSeen(messageId) {
+        async markAsSeen(lastMessageId) {
```

Comment:
Just wondering why not `message`?
I know the code right now just need the `id`, but conceptually it's easier to stay at record level when there is code at record level. Here it's a thread method that is message-dependent. So it feels more natural to have `message` in param.

Reply:
I think that in the past, there were a use where we didn't have the message, only the id